packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -65,6 +67,13 @@ class _DefaultDoctorValidatorsProvider implements DoctorValidatorsProvider {
     }
     return _validators;
   }
+
+  @override
+  List<Workflow> get workflows {
+    _workflows ??= <Workflow>[iosWorkflow, androidWorkflow];
```

Comment:
FYI - I had to change this code slightly because it introduced an issue on Windows (https://github.com/flutter/flutter/issues/21418). I've expanded this to work like the method above it, and only add the workflows to the list if they apply to the host platform.

PR: https://github.com/flutter/flutter/pull/21423